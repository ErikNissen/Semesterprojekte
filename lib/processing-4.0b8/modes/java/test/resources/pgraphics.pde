version https://git-lfs.github.com/spec/v1
oid sha256:85a89ba32c231ba37ed06b2c10f1cee63b71da249c61fdc54db1ded6b4663d11
size 106
