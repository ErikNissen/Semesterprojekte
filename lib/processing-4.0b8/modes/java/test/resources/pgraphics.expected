version https://git-lfs.github.com/spec/v1
oid sha256:7c680228b13ddb019411f3640ec0f99b8a145891cdd0f868b28badf1fa8745c3
size 770
